src/dtscalibration/datastore.py

```diff
@@ -2202,66 +2078,66 @@ def calibration_single_ended(
                 ("trans_att", time_dim),
                 ip.get_taf_pars(p_var),
             )
         else:
-            self[store_ta + "_fw"] = (("trans_att", time_dim), np.zeros((0, nt)))
-            self[store_ta + "_fw" + variance_suffix] = (("trans_att", time_dim), np.zeros((0, nt)))
+            self["talpha_fw"] = (("trans_att", time_dim), np.zeros((0, nt)))
+            self["talpha_fw_var"] = (("trans_att", time_dim), np.zeros((0, nt)))
 
-        self[store_c] = ((time_dim,), p_val[ip.c])
-        self[store_c + variance_suffix] = ((time_dim,), p_var[ip.c])
+        self["c"] = ((time_dim,), p_val[ip.c])
+        self["c_var"] = ((time_dim,), p_var[ip.c])
 
         if fix_alpha:
-            self[store_alpha] = (("x",), fix_alpha[0])
-            self[store_alpha + variance_suffix] = (("x",), fix_alpha[1])
+            self["alpha"] = (("x",), fix_alpha[0])
+            self["alpha_var"] = (("x",), fix_alpha[1])
 
         else:
-            self[store_dalpha] = (tuple(), p_val[ip.dalpha].item())
-            self[store_dalpha + variance_suffix] = (tuple(), p_var[ip.dalpha].item())
+            self["dalpha"] = (tuple(), p_val[ip.dalpha].item())
+            self["dalpha_var"] = (tuple(), p_var[ip.dalpha].item())
 
-            self[store_alpha] = self[store_dalpha] * self.x
-            self[store_alpha + variance_suffix] = (
-                self[store_dalpha + variance_suffix] * self.x ** 2
+            self["alpha"] = self.dalpha * self.x
+            self["alpha_var"] = (
+                self["dalpha_var"] * self.x ** 2
             )
 
-        self[store_ta + "_fw_full"] = (
+        self["talpha_fw_full"] = (
             ("x", time_dim),
             ip.get_taf_values(
                 pval=p_val, x=self.x.values, trans_att=self.trans_att.values, axis=""
             ),
         )
-        self[store_ta + "_fw_full" + variance_suffix] = (
+        self["talpha_fw_full_var"] = (
             ("x", time_dim),
             ip.get_taf_values(
                 pval=p_var, x=self.x.values, trans_att=self.trans_att.values, axis=""
             ),
         )
 
         tmpf = self.gamma / (
-            (np.log(self.st / self.ast) + (self[store_c] + self[store_ta + "_fw_full"]))
-            + self[store_alpha]
+            (np.log(self.st / self.ast) + (self.c + self["talpha_fw_full"]))
+            + self.alpha
         )
-        self[store_tmpf] = tmpf - 273.15
+        self["tmpf"] = tmpf - 273.15
 
         # tmpf_var
         deriv_dict = dict(
-            T_gamma_fw=tmpf / self[store_gamma],
-            T_st_fw=-(tmpf ** 2) / (self[store_gamma] * self.st),
-            T_ast_fw=tmpf ** 2 / (self[store_gamma] * self.ast),
-            T_c_fw=-(tmpf ** 2) / self[store_gamma],
-            T_dalpha_fw=-self.x * (tmpf ** 2) / self[store_gamma],
-            T_alpha_fw=-(tmpf ** 2) / self[store_gamma],
-            T_ta_fw=-(tmpf ** 2) / self[store_gamma],
+            T_gamma_fw=tmpf / self.gamma,
```

Comment:
```suggestion
            T_gamma_fw=tmpf / self["gamma"]
```
I prefer always accessing xarray variables using the ["var"] syntax, this ensures no conflicts with builtin methods etc.